### ISSUE TITLE
Missing tests

### ISSUE THREAD

**Issue by sobolevn** [achedeuzot/ueberauth_auth0#7]
Hi, Auth0 ambassador here!

I would love to help you with tests for this awesome library. I have done pretty much the same with my own lib: https://github.com/sobolevn/ueberauth_vk

But I would like to ask you before making any changes. 
Do you like my approach for testing this kind of libs? Do you have any feedback on it?

Or maybe we can come up with any other methods.
Anyway, feel free to contact me in case of any question or feedback!

P.S. I can help you maintaining this lib if you wish!

**Comment by sntran** [comment]
I welcome any tests that can ensure the correctness of this library. Personally, this library does very little, and delegates mostly to callbacks from `Überauth` and `OAuth2` libraries, so I did not think of adding tests.

I looked briefly on your test implementation, and I like the idea of using `exvcr` for recording requests and responses. I'm looking forward to seeing those tests again.

And yes, you're very welcome to maintain this library.
